platforms/Windows/bundle/installer.wixproj

```diff
@@ -31,4 +32,8 @@
   <ItemGroup Condition=" '$(INCLUDE_ARM64_SDK)' != '' ">
     <ProjectReference Include="..\sdk\sdk.wixproj" Properties="ProductArchitecture=arm64;Platform=x86" BindName="sdk_arm64" />
   </ItemGroup>
+
+  <ItemGroup Condition=" '$(INCLUDE_ANDROID_SDKS)' != '' ">
```

Comment:
Should this not be architecture specific?

Reply:
what's are the advantages of allowing to use arch-specific options/config variables instead of single switch for all android SDKs?

Reply:
or should we have just one user-facing option, but let the installer configure which SDKs to include? I guess that would make sense

Reply:
Windows allows the user to select the architectures they want to target. It does allow for some space reduction for the installed toolchain. More importantly, it is setting up support for an online installer that would reduce the overall download size.